doc/source/install.rst

```diff
@@ -24,7 +24,7 @@ at :samp:`$SRC_BASE` ( :samp:`/usr/src` by default).
 
 To install from github, run these commands:
 
-:samp:`pkg install python36 git-lite libgit2 cython3 py36-pip`
+:samp:`pkg install python37 git-lite libgit2 cython3 py37-pip`
```

Comment:
Some of these packages are no longer valid, `libgit2` is no longer needed and `cython3` has been renamed to `py37-cython`
This is a good opportunity to clean this up and make it consistent with the README.md      :smile:

Reply:
Roger that. I'll clean these up and update the PR